seastar/src/sleep.cc

```diff
@@ -19,5 +19,13 @@ VoidFuture sleep_nanos(int64_t duration) {
     co_await seastar::sleep(std::chrono::nanoseconds(duration));
 }
 
+VoidFuture lowres_sleep_secs(int32_t duration) {
+    co_await seastar::sleep<seastar::lowres_clock>(std::chrono::seconds(duration));
+}
```

Comment:
Same here - I think `lower_sleep_secs` is redundant.

Reply:
Defining duration types solved this.